### ISSUE TITLE
I am having an issue with SRA prefetch on ubuntu -Linux OS 

### ISSUE THREAD

**Issue by Nanncope** [ncbi/sra-tools#120]
Whenever I try with prefetch option given with accession number, I get  the following error message. 
I would be grateful if someone could help me to overcome this issue. 

Error msg: 
prefetch SRR2079547
Maximum file size download limit is 20,971,520KB

2018-03-30T07:25:36 prefetch.2.3.5 err: error unexpected while resolving tree within virtual file system module - failed to resolve accession 'SRR2079547' - Obsolete software. See https://github.com/ncbi/sra-tools/wiki ( 406 )

Redirected!!!

2018-03-30T07:25:38 prefetch.2.3.5 err: name incorrect while evaluating path within network system module - Scheme is 'https'
2018-03-30T07:25:38 prefetch.2.3.5 err: path not found while resolving tree within virtual file system module - 'SRR2079547' cannot be found.
Thanks in advance

**Comment by kwrodarmer** [comment]
You are using obsolete software, as the message says. The software was made obsolete mostly due to the Federal mandate to switch all communications to HTTPS, which is not supported by your version of the software.

The solution is to remove your old software and install new software. Please see our Wiki for additional information.

**Comment by Nanncope** [comment]
Thank you very much for your reply.

So it means, there is no problem with ubuntu (Linux OS). The problem is
with SRA toolkit? Is that right?

**Comment by kwrodarmer** [comment]
Yes, it means you have a version of the SRA toolkit that is guaranteed to be inoperable. You need to download the latest version (2.9.0) and install that. You'll find things work fine afterward.

**Comment by Nanncope** [comment]
Thank you very much for the information.

**Comment by xylona1900** [comment]
Hello, I had a same matter like you.
I downloaded sra toolkit-ubuntu2.9.0 from https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software
when I used prefetch, it also showed prefetch.2.3.5 err.

So, could I ask you how did you solve that problem and how to remove old software?
Thanks.

**Comment by klymenko** [comment]
Options:

1. Update your PATH environment variable after you download and unpack a new version of sratoolkit.
Then running **prefetch** command will find it in the from downloaded package.
2. Use full path to your commands - it is not convenient, you will have to always type long path.
3. You have an old version of toolkit installed on Ubuntu. Uninstall it, then use method (1) or (2) to run the current sratoolkit.

**Comment by xylona1900** [comment]
These days I find the matter is when I unzipped and then installed sratoolkit.2.9.0, it showed that the version I installed is 2.3.5. I promise I download 2.9.0 from NCBI, after unzipping I saw every command was 2.9.0 in Ubuntu, but once I inputted  **sudo apt install sra-toolkit** it would show the version.

I don't know how to deal with it.

**Comment by osris** [comment]
Ubuntu distributions have had old version of toolkit for years.  You’ll need to set a path to point to the new version you installed.


-Chris

**Comment by xylona1900** [comment]
Thanks， I resolved my problem finally!!

**Comment by Nanncope** [comment]
Hello Colleagues,

I downloaded the recent version of SRA kit using the following code: wget
http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/2.9.0/
sratoolkit.2.9.0-ubuntu64.tar.gz


I unzipped it with the follocing code:  tar xzvf
sratoolkit.2.9.0-ubuntu64.tar.gz


But how to install this unzipped folder in ubuntu OS. When I checked
the list of tools installed in ubuntu terminal using command "dpkg
--list". I cant see that sratoolkit.2.9.0 is installed. It would be
helpful if someone could let me know how to install this unzipped file
in ubuntu.


Thanks in advance
